pages/data.js

```diff
@@ -198,20 +220,30 @@ export default class Explore extends React.Component {
               <HeroContent />
               <ButtonsContainer>
                 {datasetNames.map(datasetName => (
-                  <ChangeChartButton
+                  <Link
                     key={datasetName}
-                    onClick={() => this.fetchData(datasetName)}
-                    className={
-                      datasetName === activeDataset
-                        ? 'btn btn--primary btn--chart-toggle active'
-                        : 'btn btn--primary btn--chart-toggle'
-                    }
+                    href={{
+                      pathname: '/data',
+                      query: { dataset: datasetName },
+                    }}
                   >
-                    <span className="btn--chart-toggle--icon">
-                      <img src={datasets[datasetName].icon} alt={datasets[datasetName].name} />
-                    </span>
-                    <span className="btn--chart-toggle--text">{datasets[datasetName].name}</span>
-                  </ChangeChartButton>
+                    <a
+                      onClick={() => this.fetchData(datasetName)}
+                      onKeyDown={() => this.fetchData(datasetName)}
```

Comment:
Do we need these `onClick` and `onKeyDown` handlers now that we're linking to a new path?